benchmark/bandwagon_settings.py

```diff
@@ -0,0 +1,26 @@
+"""
+Settings for bandwagon alpha.
+
+Idea:
+    We believe the 'wind' on the market.
+    The wind blow if some instruments rise/fall in continuous x day,
+    and the rate is up to y.
+    Thus, we long/short the instruments
+    following the raising/fallen of the instruments.
```

Comment:
We follow wind of instruments and positions are proportional to their accumulated change rate (in the window).